addons/mail_client_extension/controllers/main.py

```diff
@@ -170,6 +171,16 @@ def _create_company_from_iap(self, domain):
             'website': iap_data.get("domain"),
             'email': iap_data.get('email', [''])[0]
         }
+
+        logo_url = iap_data.get('logo')
+        if logo_url:
+            try:
+                response = requests.get(logo_url, timeout=2)
+                if response.ok:
+                    new_company_info['image_1920'] = base64.b64encode(response.content)
+            except Exception as e:
+                _logger.error('Download of image for new company "%s" failed error %s' % (new_company_info.name, e))
```

Comment:
logging as error is a bit strong for this, that's probably not a super critical issue if we can't get the thing. Maybe just `_logger.warning`? But if you like error it's fine by me.

Could use `%r` instead of `"%s"` though, that'll escape and quote according to Python's rules which generally works well.